### PR TITLE
Add custom resource policy flag to the enable module command

### LINKED ISSUE #1582: cli should possible manage modules CustomResourcePolicy
**Description**
By default, cli alpha `enable module` command enabled module with the default `CreateAndDelete`, if users want to use `Ignore` policy, they have to edit resources manually. It's better we allow users can control the policy through cli directly.

### PR DESCRIPTION
**Pull request by nesmabadr**
**Description**

Changes proposed in this pull request:

- Add `policy` flag to `enable module` command to manage the custom resource policy.
- Update channel and policy for already existing modules.

**Related issue(s)**
Fixes #1582

### CODE REVIEW

**Review by lindnerby — APPROVED** [review]
lgtm, waiting for tws

File: docs/gen-docs/kyma_alpha_enable_module.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Enable "my-module" from "alpha" channel in "default-kyma" in "kyma-system" Names
   -f, --force-conflicts    Forces the patching of Kyma spec modules in case their managed field was edited by a source other than Kyma CLI.
   -k, --kyma-name string   Kyma resource to use. If empty, 'default-kyma' is used. (default "default-kyma")
   -n, --namespace string   Kyma Namespace to use. If empty, the default 'kyma-system' Namespace is used. (default "kyma-system")
+  -p, --policy string      Determines how the module is managed, either install or ignore the default values provided in the module template. Available values are 'CreateAndDelete' or 'Ignore'. (default "CreateAndDelete")
```
Comment (nataliasitko) [inline docs/gen-docs/kyma_alpha_enable_module.md:39]:
```suggestion
  -p, --policy string      Determines how the module is managed. Use 'CreateAndDelete' to install the default values provided in the module template or 'Ignore' to ignore them. (default "CreateAndDelete")
```

Comment (nesmabadr) [reply docs/gen-docs/kyma_alpha_enable_module.md]: Changed.

File: cmd/kyma/alpha/enable/module/opts.go [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
 import (
+	"fmt"
 	"time"
 
 	"github.com/kyma-project/cli/internal/cli"
 	"github.com/pkg/errors"
 )
 
+const (
+	CustomResourcePolicyCreateAndDelete = "CreateAndDelete"
```
Comment (lindnerby) [inline cmd/kyma/alpha/enable/module/opts.go:12]: Since these consts are not used outside of it (for now), I guess they can be package-private.

**Review by nataliasitko — APPROVED** [review]

File: cmd/kyma/alpha/enable/module/opts.go [hunk 3]
```diff
@@ -1,19 +1,26 @@
...
 import (
+	"fmt"
 	"time"
 
 	"github.com/kyma-project/cli/internal/cli"
 	"github.com/pkg/errors"
 )
 
+const (
+	customResourcePolicyCreateAndDelete = "CreateAndDelete"
```
Comment (jakobmoellersap) [inline cmd/kyma/alpha/enable/module/opts.go:12]: can you reference the existing v1beta1 API from lifecycle manager here? Its included as an in-code dependency already.

**Review by jakobmoellersap — CHANGES_REQUESTED** [review]
Mini-Nit, other than that LGTM :) Also needs TW review after that change again

**Review by lindnerby — APPROVED** [review]

**Review by jakobmoellersap — APPROVED** [review]

**Review by nataliasitko — APPROVED** [review]

**Pull request merged**